### PR TITLE
Blob compression

### LINKED ISSUE #4: Compression Algorithm
We should choose a proper compression algorithm.

Will edit these with possible options.

### PR DESCRIPTION
**Pull request by orkunkilic**
Compressing/Decompressing blobs with Brotli

Needs e2e test in a demo rollup

Closes #4

### CODE REVIEW

**Comment by orkunkilic** [comment]
Demo rollup is working with compression

File: src/helpers/builders.rs [hunk 1]
```diff
@@ -31,6 +32,18 @@ pub fn get_satpoint_to_inscribe(utxo: &UTXO) -> SatPoint {
     SatPoint::from_str(&satpoint_str).unwrap()
 }
 
+pub fn compress_blob(blob: &[u8]) -> Vec<u8> {
+    let mut writer = CompressorWriter::new(Vec::new(), 4096, 11, 22);
```
Comment (eyusufatik) [inline src/helpers/builders.rs:36]: could you explain where these numbers come from?

buffer size -> 4096, why 4096 what does it effect
quality (?) -> 11
lgwin? -> 22

**Review by eyusufatik — APPROVED** [review]
LGTM

File: src/helpers/builders.rs (hunk 1 above)
Comment (orkunkilic) [reply src/helpers/builders.rs:36]: quality -> compression ratio. 11 is max. max but slower compression.
lgwin -> window size. 24 max.  big window size = big memory. sequencer does not have mem problem so.
buffer size -> I think it is batch size to evaluate in single round.

This was the most common setting I have seen. will look again.

Comment (orkunkilic) [reply src/helpers/builders.rs:36]: looks like it is what I said. brotli defaults to 4096. merging with this conf.

**Pull request merged**